ui/raidboss/emulator/data/network_log_converter/LineEvent0x03.ts

```diff
@@ -36,19 +79,19 @@ export class LineEvent0x03 extends LineEvent {
     });
   }
 
-  convert() {
+  convert(): void {
     let CombatantName = this.name;
     if (this.server !== '')
       CombatantName = CombatantName + '(' + this.server + ')';
 
-    this.convertedLine = this.prefix() +
-      this.id.toUpperCase() + ':' +
-      'Added new combatant ' + CombatantName +
-      '.  Job: ' + this.jobName +
-      ' Level: ' + this.level +
-      ' Max HP: ' + this.maxHp +
-      ' Max MP: ' + this.maxMp +
-      ' Pos: (' + this.parts[17] + ',' + this.parts[18] + ',' + this.parts[19] + ')';
+    this.convertedLine = `\
+${this.prefix()}${this.id.toUpperCase()}:\
+Added new combatant ${CombatantName}.\
+  Job: ${this.jobName}\
+ Level: ${this.level}\
+ Max HP: ${this.maxHp}\
+ Max MP: ${this.maxMp}\
+ Pos: (${this.parts[17]},${this.parts[18]},${this.parts[19]})`;
```

Comment:
```suggestion
    this.convertedLine = this.prefix() +
      this.id.toUpperCase() + ':' +
      'Added new combatant ' + CombatantName +
      '.  Job: ' + this.jobName +
      ' Level: ' + this.level.toString() +
      ' Max HP: ' + this.maxHp.toString() +
      ' Max MP: ' + this.maxMp.toString() +
      ' Pos: (' + this.parts[17] + ',' + this.parts[18] + ',' + this.parts[19] + ')';
```

I find this multiline template literal form to not be very readable as compared to the original code.  (Mostly it's the indentation that's very jarring to my eyes?) What about using the previous code with `toString()` attached to anything that's not a string?

Reply:
I have no strong opinions either way on this one, I can swap back to string appending instead. I assume you want this adjustment for all the line events for consistency?

Reply:
Yeah, that would be great, thanks!